processing/src/main/java/io/druid/query/filter/SpatialDimFilter.java

```diff
@@ -33,11 +34,13 @@
 public class SpatialDimFilter implements DimFilter
 {
   private final String dimension;
+  private final String delimiter;
```

Comment:
Shouldn't it be part of `equals()`, `hashCode()` and `toString()` of this class?